types/lesgo/middlewares/verifyJwtMiddleware.d.ts

```diff
@@ -0,0 +1,12 @@
+import { MiddlewareFunction } from "middy";
```

Comment:
The package has no this native dependency:
https://github.com/reflex-media/lesgo-framework/blob/master/src/middlewares/verifyJwtMiddleware.js

Reply:
@peterblazejewicz The use case for this is actually like in this example here:
https://reflex-media.github.io/lesgo-docs/basics/middleware/#http in the `Usage` section

all of the middlewares just returns a function for use in middy